stdlib/test/collections/test_set.mojo

```diff
@@ -15,6 +15,7 @@
 from collections import Set
 
 from testing import assert_false, assert_raises, assert_true
+from testing import assert_equal as AE
```

Comment:
**Suggestion** I think it would be better for consistency and readability if we used this as `assert_equal` instead of using a custom alias. I understand the desire for brevity in a common function like this, but I think we should weight consistently higher.

Reply:
I actually did that since this file defines it's own `assert_equal` that drops the `Stringable` requirement which I don't personally like but I didn't want to go changing all the tests

Reply:
Ah I see, thanks for clarifying. That's not ideal, but I won't hold this PR up on that. Thanks for adding this!

Reply:
On second thought I've gone and just renamed the local function to make it more clear in the future.